src/recipes/editor-plugin/inspector-plugins.md

```diff
@@ -0,0 +1,342 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Inspector plugins
+
+The inspector dock allows you to create custom widgets to edit properties through plugins.
+This can be beneficial when working with custom datatypes and resources, although you can
+use the feature to change the inspector widgets for built-in types. You can design custom
+controls for specific properties, entire objects, and even separate controls associated
+with particular datatypes.
+For more info, see
+[docs.godotengine.org](https://docs.godotengine.org/en/stable/classes/class_editorinspectorplugin.html#class-editorinspectorplugin).
+
+The [example](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/inspector_plugins.html) in the Godot docs in Rust.
+
+Before:
+
+![Before](./images/before.png)
+
+After:
+
+![After](./images/after.png)
```

Comment:
The first line is not a full sentence.

Also, maybe say something about the before/after images. For example, what you are going to do between them.

Reply:
I can add description. If you will write like this:
```
After:

![After](./images/after.png)
```
for markdown it will like you have one line - text and image will be on one line

Reply:
If there's an empty line in Markdown in between, it should do a paragraph (HTML `<p> </p>`), no?

Reply:
only if two empty lines
otherwise, how do we split a long text that is limited to 150 characters into several lines and it saves the paragraph?